pycode/memilio-simulation/memilio/simulation/secir.cpp

```diff
@@ -18,15 +18,23 @@
 * limitations under the License.
 */
 
-#include "templates.h"
+#include "pybind_util.h"
+#include "compartments/simulation.h"
+#include "compartments/compartmentalmodel.h"
+#include "epidemiology/populations.h"
+#include "utils/custom_index_array.h"
+#include "utils/parameter_set.h"
+#include "utils/index.h"
+#include "mobility/graph_simulation.h"
+#include "mobility/mobility.h"
 #include "secir/secir.h"
 #include "secir/parameter_studies.h"
 #include "secir/analyze_result.h"
 #include "Eigen/Core"
 #include "pybind11/stl_bind.h"
 #include <vector>
 
-namespace py = pybind11;
+using namespace pybind11;
```

Comment:
same as in simulation.cpp, ` using namespace pybind11` and qualified names `pybind11::...` is useless.